### PR TITLE
Add public options to event location, and add host hash

### PR DESCRIPTION
**Pull request by ben-pr-p**

### CODE REVIEW

**Comment by j-ro** [comment]
Hi Ben, thanks! Generally you'll want to join our tech calls to discuss getting stuff like this merged into the spec.  Events already have a creator link, for instance, which is more appropriate than a host hash. And the public or not boolean is already present in the visibility field.

**Comment by joshco** [comment]
@ben-pr-p status update or ETA?

**Comment by joshco** [comment]
Next steps
Rename hash to "contact"
add "additional_info" free text string

**Comment by ben-pr-p** [comment]
Should it be `contact.email` or `contact.email_address`? The latter is more verbose but I think I've seen it preferred elsewhere

**Comment by j-ro** [comment]
email_address is preferred I believe, yes, phone_number too

**Comment by jlegrone** [comment]
Deployed changes: https://brandnewcongress.github.io/osdi-docs/events.html#contact

**Comment by j-ro** [comment]
Looks good to me!

File: events.md [hunk 1]
```diff
@@ -111,6 +112,17 @@ These JSON hashes included in the table above are broken out into their own tabl
...
+|location.public |boolean |Whether the venue's location should be shared publicly, or if false, only shared with RSVPs (for example, someone's house)
+
+#### Contact
+
+|Name          |Type      |Description
+|-----------    |-----------|--------------
+|contact.name	|string	|Name of the host or contact person for event (e.g., Jane Doe)
+|contact.email_address	|string	|Email address of the host (jane.doe@hotmail.co.uk)
+|contact.phone_number	|string	|Phone number of the host (214-555-0869)
+|contact.additional_info |string  |Free form place for information about the event contact
```
Comment (jlegrone) [inline events.md]: Two small nitpicks:

The convention for descriptions seems to be to end with a period `.` and add any example value in parens after that.

```
A human readable identifier of the system where this event was created. (ex: “OSDI System”)
```

Additionally, we might consider adding `May contain text and/or HTML.` to the description for `contact.additional_info` (similar to the `instructions` field above).

Comment (j-ro) [reply events.md]: Do implementations out there contain HTML in that field? I know for us (and we're the ones who wanted that field) it's all plain text...

**Comment by j-ro** [comment]
approved 9/28

**Pull request merged**